integreat_cms/cms/templates/events/event_list.html

```diff
@@ -130,6 +130,9 @@ <h1 class="heading">{% trans 'Events' %}</h1>
                         </option>
                     {% endif %}
                 {% endif %}
+                <option data-bulk-action="{% url 'simple_german_translation_pages' %}" title=" {% trans 'Automatically translate this page to easy language' %}">
+                    {% trans 'Automatically translate this page to easy language' %}
+                </option>
```

Comment:
I guess it makes sense to show this option only when the `SUMM_AI_ENABLED` setting is `True`, right?
The same goes for the other content types.